File: RYG led control.py
#RED=15
#YELLOW=16
#GREEN=18
import sys
import time
import RPi.GPIO as GPIO ## Import GPIO library
GPIO.setwarnings(False)
GPIO.setmode(GPIO.BOARD) ## Use board pin numbering
GPIO.setup(15, GPIO.OUT) ## Setup GPIO Pin 15 to OUT
GPIO.setup(16, GPIO.OUT) ## Setup GPIO Pin 16 to OUT
GPIO.setup(18, GPIO.OUT) ## Setup GPIO Pin 18 to OUT
print(len(sys.argv))
        
status=sys.argv[2]

if(status=="ron"):
    GPIO.output(15,True)
if(status=="roff"):
    GPIO.output(15,False)

if(status=="yon"):
    GPIO.output(16,True)
if(status=="yoff"):
    GPIO.output(16,False)

if(status=="gon"):
    GPIO.output(18,True)
if(status=="goff"):
    GPIO.output(18,False)

File: RYG led control.php
<!DOCTYPE html>
<html>
	
<?php
#$output = shell_exec("sudo python /var/www/html/myRaspv1.0.py ".strtolower($_GET['red action', 'yellow action','green action'])) ;
if (!empty($_GET["action"])) {
  $output = shell_exec("sudo python3 /var/www/html/test.py Red_led ".strtolower($_GET["action"])) ;
}
?>

<form action="<?php echo $_SERVER['PHP_SELF'];?>" method="get">
<label>RED LED STATUS:
On <input type="radio" name="action" value="ron" onchange="javascript:submit()"/> 
Off <input type="radio" name="action" value="roff" onchange="javascript:submit()"/>
</label>

<label>YELLOW LED STATUS:
On <input type="radio" name="action" value="yon" onchange="javascript:submit()"/> 
Off <input type="radio" name="action" value="yoff" onchange="javascript:submit()"/>
</label>

<label>GREEN LED STATUS:
On <input type="radio" name="action" value="gon" onchange="javascript:submit()"/> 
Off <input type="radio" name="action" value="goff" onchange="javascript:submit()"/>
</label>

</form>
</html>
